lib/osf-components/addon/components/search-bar/template.hbs

```diff
@@ -9,10 +9,12 @@
                 @fixedWidth={{true}}
             />
         </span>
+        <label for='search' local-class='displayNone'>{{t 'osf-components.search-bar.search'}}</label>
```

Comment:
I get that this lets us pass a11y linting, but if we're not displaying the label I think we'd be better off leaving it off. If we're not meeting the spirit of the a11y guidelines, I'd rather not bend around to make it look like we are.